packages/atomic/src/components/insight/atomic-insight-search-box/atomic-insight-search-box.tsx

```diff
@@ -0,0 +1,86 @@
+import {Component, h, Prop, State} from '@stencil/core';
+import {buildInsightSearchBox} from '@coveo/headless/insight';
+import {SearchBox, SearchBoxState} from '@coveo/headless';
+import {randomID} from '../../../utils/utils';
+import {
+  BindStateToController,
+  InitializeBindings,
+} from '../../../utils/initialization-utils';
+import {Bindings} from '../atomic-insight-interface/atomic-insight-interface';
+import {SearchInput} from '../../common/search-box/SearchInput';
+import {SearchBoxWrapper} from '../../common/search-box/SearchBoxWrapper';
+import {SubmitButton} from '../../common/search-box/SubmitButton';
+
+/**
+ *
+ * @internal
+ */
+@Component({
+  tag: 'atomic-insight-search-box',
```

Comment:
What's the naming scheme we want to adhere to ?

* `atomic-insight-interface`, `atomic-insight-search-box`, etc.

* `atomic-svc-insight-interface`, `atomic-svc-insight-search-box` etc.

* `atomic-svc-interface`, `atomic-svc-search-box`, etc.

* Other ?

Reply:
I think that, following headless, `atomic-insight-search-box` and `atomic-insight-interface` make sense and is easier to type compared to needing the extra `svc` there